test/merchant_test.rb

```diff
@@ -40,6 +35,11 @@ def test_find_invoices_returns_invoices_associated_with_merchant
     assert_equal 1, result.count
   end
 
+  def test_revenue_reports_all_revenue_for_a_merchant
+    result = @merchant_repository.merchants[9].revenue
+    assert_equal BigDecimal.new('1872.74'), result
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.